eve/workers/pod.yaml

```diff
@@ -73,7 +73,7 @@ spec:
         - -ec
         - |
           # wait for exports to be created
-          sleep 10
+          sleep 15
           source /root/.aws/exports || exit 1
```

Comment:
Wouldn't it be better to loop every second until this succeeds instead to reduce flakiness even further without having to wait too long in most cases? (with a timeout, say e.g. 60 seconds)

Reply:
I had this originally but there were quirks with kubernetes that made it so whenever the s3 container accessed the file first (through shared volumes), it would never find `/root/.aws/exports` even after file creation. I'll try it again but I know Rel-eng is working on a better way to pass the environment variables through from eve to the container which would remove the need for the "sleep && source /root/.aws/exports" entirely.

Reply:
Indeed! I'm currently working on removing this work around.
I will open a PR as soon as I finish the work. :)